AWS_CloudWatch/FIleMetricPublisher.py

```diff
@@ -0,0 +1,79 @@
+from Functions.MetricPublisher import MetricPublisher
+from Functions.Metrics import CounterMetric
+import unittest
+from datetime import datetime
+import json
+import os
+import time
+
+
+class FileMetricPublisher(MetricPublisher):
+    def __init__(self,file_location:str,name_space:str):
+        self.file_location = file_location
```

Comment:
private member should be named with a preceding _

Reply:
btw are you running the linter and formatter? Check with you colleagues, this should be happening by default due to the git hooks @yydiamen @ygins